src/styles/home.styl

```diff
@@ -158,7 +157,8 @@ a.signin-link {
   width: 50%;
   width: 90vh;
   squareify();
-  background: desaturate(darken($colorPrimary, 30%), 20%);
+  // background: desaturate(darken($colorPrimary, 30%), 20%);
```

Comment:
No need to leave these in here commented out I don't think. We've decided that this new look is better